src/App.scss

```diff
@@ -36,6 +36,9 @@ body {
         padding: 0 60px;
     }
 }
+.wrapper-space{
```

Comment:
space missing

Reply:
fixed

Reply:
not fixed

Reply:
it is strange due to I have this changes but here isn't and on git branch command, my branch is absent
![image](https://user-images.githubusercontent.com/41084902/115667415-7cb5d500-a34e-11eb-863c-c72e0dd95a91.png)